locales/en-US/strings.properties

```diff
@@ -1,29 +1,39 @@
-top_sites=Top Sites
-highlights=Highlights
-visited=Visited
-bookmarked=Bookmarked
-synced=Synced from another device
-open=Open
-topic=Topic
-bookmark=Bookmark
-remove_bookmark=Remove Bookmark
-share=Share
-copy_address=Copy Address
-email_link=Email Link...
-open_new_window=Open in a New Window
-open_private_window=Open in a Private Window
-dismiss=Dismiss
-delete=Delete from History
+header_top_sites=Top Sites
+header_highlights=Highlights
+type_label_visited=Visited
```

Comment:
At minimum add a empty line here to separate `header` from `type_label`. We should probably have a note describing what a "type_label" is used for, e.g., "These labels are shown with a page to explain how it's related to the user."